ui/components/GrafanaComponent.js

```diff
@@ -102,7 +102,7 @@ class GrafanaComponent extends Component {
         }
         if (name === 'grafanaBoardSearch') {
             if (this.boardSearchTimeout) clearTimeout(this.boardSearchTimeout);
-            this.boardSearchTimeout = setTimeout(this.getGrafanaBoards, 500); // to delay the search by a few.
+            this.boardSearchTimeout = setTimeout(this.getGrafanaBoards, 1000); // to delay the search by a few.
```

Comment:
thanks, @ashiscs. Do you know if 1000ms is generally long enough of a timeout period? Will you be creating another PR to incorporate display of the waiting indicator?

@girishranganathan what design pattern do we want to use for timeouts? Should we sidecar an envoy to Meshery's deployment and use Envoy to control this resiliency logic?

Reply:
Ok sir!